src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -0,0 +1,343 @@
+        File workspaceDir = new File(workspace);
+
+        if (overwriteExistingWorkspace) {
+            IOUtils.tryDelete(workspaceDir);
+        }
+
+        int ret = GenomicsDBImporter.createTileDBWorkspace(workspaceDir.getAbsolutePath());
+
+        if (ret > 0) {
+            File tempFile = new File(workspaceDir.getAbsolutePath() + "/__tiledb_workspace.tdb");
+            if (!tempFile.exists()) {
+                throw new UserException(workspaceDir.getAbsolutePath() + " is not a valid GenomicsDB workspace");
+            }
+
+            logger.info("Importing data to GenomicsDB workspace: " + workspaceDir);
+        } else if (ret < 0) {
+            throw new UserException("Error creating GenomicsDB workspace: " + workspaceDir);
+        }
+
+        return workspaceDir;
+    }
+
+    /**
+     * A complete traversal from start to finish. This method will import all samples
+     * specified in the input GVCF files.
+     */
+    @Override
+    public void traverse() {
+        try {
+            importer.importBatch();
+        } catch (IOException e) {
+            throw new UserException("GenomicsDB importBatch raised exception: " + e.getMessage());
+        }
+    }
+
+    @Override
+    public Object onTraversalSuccess() {
+        logger.info("Import completed!");
+        return importer.isDone();
+    }
+
+    @Override
+    public void closeTool() {
+        for (Map.Entry<String, FeatureReader<VariantContext>> reader : sampleToVCFMap.entrySet()) {
+            try {
+               reader.getValue().close();
+            } catch (IOException e) {
+                throw new GATKException("FeatureReader close() for " + reader.getKey() +
+                    " has thrown an IOException: " + e.getMessage());
+            }
+        }
+    }
+
+    /**
+     * Before traversal starts, create the feature readers
+     * for all the input GVCFs, create the merged header and
+     * initialize the interval
+     */
+    @Override
+    public void onStartup() {
+
+        for (String variantPath : variantFilePaths) {
+
+            final String variantURI = IOUtils.getPath(variantPath).toAbsolutePath().toUri().toString();
+            final Function<SeekableByteChannel, SeekableByteChannel> cloudWrapper = (cloudPrefetchBuffer > 0 ? is -> SeekableByteChannelPrefetcher.addPrefetcher(cloudPrefetchBuffer, is) : Function.identity());
+            final Function<SeekableByteChannel, SeekableByteChannel> cloudIndexWrapper = (cloudIndexPrefetchBuffer > 0 ? is -> SeekableByteChannelPrefetcher.addPrefetcher(cloudIndexPrefetchBuffer, is) : Function.identity());
+            AbstractFeatureReader<VariantContext, LineIterator> reader = AbstractFeatureReader.getFeatureReader(variantURI, null, new VCFCodec(), true, cloudWrapper, cloudIndexWrapper);
+
+            // A GVCF file must contain only one sample, throw an exception otherwise
+            if (((VCFHeader)reader.getHeader()).getGenotypeSamples().size() > 1) {
```

Comment:
Also change `> 1` to `!= 1` -- ie., it should be an error to either have multiple samples or no samples. Update error message accordingly.